Packs/CommonScripts/Scripts/ArrayToCSV/ArrayToCSV_test.py

```diff
@@ -0,0 +1,43 @@
+from ArrayToCSV import arr_to_csv_command
+
+
```

Comment:
Note that these cases are basically the same test case with different inputs and output, and its probably better to do it with "parametrize" next time.